src/utils/functions/cutTextByWords.ts

```diff
@@ -1,4 +1,9 @@
 /* eslint-disable import/prefer-default-export */
-export const cutTextByWords = (text: string, countWords: number) => {
-  return `${text.split(" ").slice(0, countWords).join(" ")}...`;
+export const cutTextByWords = (text: string | null, countWords: number) => {
```

Comment:
I don't think this function is necessary, try doing this with CSS.  [For example](https://stackoverflow.com/questions/11426275/how-can-i-show-dots-in-a-span-with-hidden-overflow#:~:text=Quite%20easy%20thing%20to%20do,with%20text%2Doverflow%3Aellipsis.)

Reply:
wow, thank you